src/Playwright/Core/Locator.cs

```diff
@@ -40,18 +40,17 @@ internal class Locator : ILocator
 {
     internal readonly Frame _frame;
     internal readonly string _selector;
+    private static readonly JsonSerializerOptions _locatorSerializerOptions = new()
```

Comment:
Should statics be at the top of class?

Reply:
I was following the style of the following `_testIdAttributeName`, but if that's the preferred code style I can move this field as well as `_testIdAttributeName` to the top